scania-digital-design-system/tegel
# fix(storybook): added missing comma to controls

**Describe pull-request**  
A missing comma in controls breaks the build

**Solving issue**  
Fixes: Was introduced [here](https://github.com/scania-digital-design-system/tegel/pull/516/files#diff-2ecaa36781fcd043db683b366413d383cce44ac0cdd878f719dc13326009332cR91)

**How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to develop branch and open console. 
Check below screenshots for error:
<img width="1345" alt="Skärmavbild 2024-02-20 kl  22 31 56" src="https://github.com/scania-digital-design-system/tegel/assets/201671/6e602d47-3467-4e87-947d-1dca71d7ded2">
<img width="1308" alt="Skärmavbild 2024-02-20 kl  22 31 36" src="https://github.com/scania-digital-design-system/tegel/assets/201671/be076a79-b8cc-47fe-ac50-d40962de6ee0">
